Lib/unittest/__main__.py

```diff
@@ -13,6 +13,6 @@
 
 __unittest = True
 
-from .main import main, TestProgram
```

Comment:
Lib/unittest/main.py ends with "main = TestProgram". And TestProgram is not used in Lib/unittest/__main__.py.